### PR TITLE
1.16 alchemy recipes

### PR DESCRIPTION
**Pull request by Dweblenod**
Adding recipes for the new items/blocks added to nether update

### CODE REVIEW

**Comment by Dweblenod** [comment]
Additional stuff: Reduced cost of red mushrooms(now closer in universal value to brown), and added banner costs for uncraftable varieties

**Comment by Dweblenod** [comment]
Netherite is intentionally being left without a combination recipes(has a cost) to encourage nether exploration and potential rare find chance in land dungeons in the future

File: src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java [hunk 1]
```diff
@@ -220,6 +226,17 @@ protected void buildShapelessRecipes(Consumer<IFinishedRecipe> consumer)
...
 		CombinationRecipeBuilder.of(Items.GLOWSTONE).input(Items.NETHERRACK).and().input(Items.GLOWSTONE_DUST).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.CRIMSON_STEM).input(ItemTags.LOGS).or().input(Items.CRIMSON_FUNGUS).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.WARPED_STEM).input(ItemTags.LOGS).or().input(Items.WARPED_FUNGUS).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.CRIMSON_ROOTS).input(Items.GRASS).or().input(Items.CRIMSON_FUNGUS).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.WARPED_ROOTS).input(Items.GRASS).or().input(Items.WARPED_FUNGUS).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.WEEPING_VINES).input(Items.VINE).and().input(Items.CRIMSON_ROOTS).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.TWISTING_VINES).input(Items.VINE).and().input(Items.WARPED_ROOTS).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.CRIMSON_NYLIUM).input(Items.MYCELIUM).or().input(Items.CRIMSON_NYLIUM).buildFor(consumer, Minestuck.MOD_ID);
```
Comment (kirderf1) [inline src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java:235]: I'd like to bring attention to this recipe, which looks unintentional

Comment (Dweblenod) [reply src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java]: Crimson nylium recipe should be fixed!

**Pull request merged**